{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/docusignapi\/reachabilityerror\/unabletosetcallback"]}],"hierarchy":{"paths":[["doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI","doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI\/ReachabilityError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"UnableToSetCallback","kind":"identifier"}],"languages":["swift"]}]}],"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI\/ReachabilityError\/UnableToSetCallback"},"metadata":{"symbolKind":"case","title":"ReachabilityError.UnableToSetCallback","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"UnableToSetCallback"}],"role":"symbol","externalID":"s:11DocuSignAPI17ReachabilityErrorO19UnableToSetCallbackyA2CmF","roleHeading":"Case","modules":[{"name":"DocuSignAPI"}]},"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://com.DocuSign.DocuSIgnAPI/documentation/DocuSignAPI":{"kind":"symbol","type":"topic","role":"collection","url":"\/documentation\/docusignapi","abstract":[],"title":"DocuSignAPI","identifier":"doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI"},"doc://com.DocuSign.DocuSIgnAPI/documentation/DocuSignAPI/ReachabilityError":{"role":"symbol","type":"topic","url":"\/documentation\/docusignapi\/reachabilityerror","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"ReachabilityError","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI\/ReachabilityError","navigatorTitle":[{"text":"ReachabilityError","kind":"identifier"}],"abstract":[],"title":"ReachabilityError"},"doc://com.DocuSign.DocuSIgnAPI/documentation/DocuSignAPI/ReachabilityError/UnableToSetCallback":{"type":"topic","identifier":"doc:\/\/com.DocuSign.DocuSIgnAPI\/documentation\/DocuSignAPI\/ReachabilityError\/UnableToSetCallback","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"UnableToSetCallback"}],"url":"\/documentation\/docusignapi\/reachabilityerror\/unabletosetcallback","title":"ReachabilityError.UnableToSetCallback","kind":"symbol","role":"symbol","abstract":[]}}}